user_guide_src/source/helpers/array_helper.rst

```diff
@@ -57,6 +57,23 @@ The following functions are available:
         // Returns: 23
         $baz = dot_array_search('foo.*.baz', $data);
 
+    If the array key contains a dot, then the key can be escaped with a backslash::
+        $data = [
```

Comment:
Please fix the check warnings in the user guide.

Reply:
Thanks. Fixed.